### DISCUSSION TITLE
[User] Insert summary of your issue or enhancement..

### DISCUSSION THREAD

**Discussion by Lzhang-hub** [ggml-org/llama.cpp#3700] [Q&A]
# Expected Behavior
For a prompt will generate 1000 output tokens, is there function to stop generate token manualy?

**Comment by shibe2** [comment]
How do you start generation?

**Reply by Lzhang-hub** [reply]
I run with command:  `./server -m model/ggml-model.gguf -c 2048`, and then post request.

**Reply by shibe2** [reply]
With streaming (`stream: true` in the request), when you received enough text, close the connection.
